File: Python/453. Minimum Moves to Equal Array Elements.py
"""
ans = tag - min(nums)
(n-1)*ans = sum(tag-num)
"""
class Solution:
    def minMoves(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        mi = nums[0]
        for num in nums:
            mi = min(mi, num)
        return sum(nums) - len(nums)*mi

File: C++/453. Minimum Moves to Equal Array Elements.cpp
class Solution {
public:
    int minMoves(vector<int>& nums) {
        int ans = nums[0],res = 0;
        int len = nums.size();
        for(auto num: nums)
            ans = min(num, ans);
        for(int i=0;i<len;i++)
            res += nums[i]-ans;
        return res;
    }
};